dolphinscheduler-data-quality/src/main/java/org/apache/dolphinscheduler/data/quality/utils/StringUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.data.quality.utils;
+
+/**
+ * StringUtils
+ */
+public class StringUtils {
```

Comment:
Why not add the dependency of `dolphinscheduler-common`, so that you can use some utils directly.

Reply:
The data quality module is an independent module. I don’t want to introduce dolphinscheduler-common in it. The purpose 1 is to reduce the dependence on the code in the dolphinscheduler so that the data-quality can be separated in the future. The purpose 2 is to make the data-quality as lightweight as possible.